pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[build-system]
+requires = [
+    "setuptools>=80.9.0",
```

Comment:
setuptools 80.9.0 require a higher version than devlib: 
> Requires: Python >=3.9 

https://pypi.org/project/setuptools/80.9.0/

However, I'm not so sure devlib actually works with 3.7 anymore, and Ubuntu 20.04 (focal) has reached EOL earlier this year. So the next LTS is Jammy 22.04, which ships Python 3.10.

As a result, I think it's acceptable to bump the required Python version in devlib to 3.9 at least. I think this was discussed on another PR but I can't remember which one.

@marcbonnici Any thoughts ?